Specs/NumberWithUnit/English/CurrencyModel.json

```diff
@@ -2201,5 +2201,38 @@
         }
       }
     ]
+  },
+  {
+    "Input": "10 turkish lira 11a",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "10 turkish lira",
+        "Start": 10,
+        "End": 24,
+        "TypeName": "currency",
+        "Resolution": {
+          "unit": "Turkish lira",
+          "value": "10"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "deposits: 10 turkish lira 1111-22223333-4444",
```

Comment:
Can you add this case in Turkish?
"mustafa ustaya 10 bin lira 1111-22223333-4444 nolu hesaba havale yap"